takoyaki/src/main/java/frc/robot/Constants.java

```diff
@@ -4,12 +4,21 @@
 
 package frc.robot;
 
-/**
- * The Constants class provides a convenient place for teams to hold robot-wide numerical or boolean
- * constants. This class should not be used for any other purpose. All constants should be declared
- * globally (i.e. public static). Do not put anything functional in this class.
- *
- * <p>It is advised to statically import this class (or one of its inner classes) wherever the
- * constants are needed, to reduce verbosity.
- */
-public final class Constants {}
+import edu.wpi.first.wpilibj.XboxController;
+
+public final class Constants {
+    public static int
+        climbUp = XboxController.Button.kY.value,
```

Comment:
all constants are should be public static final and all caps with an underscore between words

Reply:
yes, please do not instantiate multiple at a time, please do one at a time.

e.g.

```
public static final int CLIMB_UP = XboxController.Button.kY.value;
public static final int CLIMB_DOWN = XboxController.Button.kA.value;
```

Reply:
Even better if you can name them something like `CLIMB_UP_BUTTON`